### PR TITLE
fix: reduce buttons

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Collapse the buttons introduced by #142 .

## How Has This Been Tested?

- The click event will be executed correctly.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -199,20 +199,34 @@ <h2>最新バージョン</h2>
...
+										btn btn-primary
+										mb-2
+										dropdown-toggle dropdown-toggle-split
+									"
+									data-bs-toggle="dropdown"
+									aria-expanded="false"
+								>
+									<span class="visually-hidden">Toggle Dropdown</span>
+								</button>
+								<ul class="dropdown-menu">
```
Comment (hal-shu-sato) [inline src/index.html:222]: How about this?
It makes the menu to right align.
https://getbootstrap.com/docs/5.1/components/dropdowns/#menu-alignment
```suggestion
								<ul class="dropdown-menu dropdown-menu-end">
```

**Review by hal-shu-sato — COMMENTED** [review]
I have one suggestion.

**Comment by mitosagi** [comment]
Fixed the problems pointed out.

Also added a fix for the error message being hidden in the toggle.

**Review by hal-shu-sato — APPROVED** [review]
Great!

**Pull request merged**